### ISSUE TITLE
DNS Over HTTPS pop in recording

### ISSUE THREAD

**Issue by cmdavinder** [aws-samples/amazon-chime-sdk-recording-demo#6]
Hi there,

All recorded meetings seem to have "DNS over HTTPs" warning. Is it just me or it is a known issues? See below screenshot

<img width="396" alt="Screen Shot 2020-04-21 at 12 28 24 AM" src="https://user-images.githubusercontent.com/22786637/79825516-31499f00-8367-11ea-8bfe-3b8c55a12ddb.png">

**Comment by peternijssen** [comment]
We are having the same problem on our end. 

Just very recent, I've found a potential solution, but haven't been able to try it.
add:
```
user_pref("doh-rollout.doorhanger-shown", true);
```

to the following lines:
https://github.com/aws-samples/amazon-chime-sdk-recording-demo/blob/master/recording/run.sh#L41

This is based on the following lines within the code of Mozilla:
```
  async rememberDoorhangerShown() {
    // This will be shown on startup and network changes until a user clicks
    // to confirm/disable DoH or presses the esc key (confirming)
    log("Remembering that doorhanger has been shown");
    await rollout.setSetting(DOH_DOORHANGER_SHOWN_PREF, true);
  },
```
@ https://hg.mozilla.org/releases/mozilla-release/file/tip/browser/extensions/doh-rollout/background.js#l111

Again, we haven't tried this yet. Some other settings within the file above are interesting. Also, it doesn't stop any future doorhangers from being displayed, just potentially solves only this doorhanger.

**Comment by cmdavinder** [comment]
Your suggestion works! @peternijssen

**Comment by nilaydave38** [comment]
I had the same issue even audio also not recorded.

Anyone facing an audio recording issue as well?

**Comment by anuranduttaroy** [comment]
This should be fixed with the [PR#10](https://github.com/aws-samples/amazon-chime-sdk-recording-demo/pull/10)
